.travis.yml

```
@@ -64,6 +64,9 @@ before_script:
     # Make sure stdout is in blocking mode. Otherwise builds will fail due to large writes to stdout
     # See https://github.com/travis-ci/travis-ci/issues/4704. If this gets fixed, this line can also be removed.
     - python3 -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); fcntl.fcntl(sys.stdout, fcntl.F_SETFL, flags&~os.O_NONBLOCK);'
+    # Build docker image only for develop branch of the main repo
+    - if [ "$TRAVIS_REPO_SLUG" != "dashpay/dash" -o "$TRAVIS_BRANCH" != "develop" -o "$TRAVIS_PULL_REQUEST" = "true" ]; then export DOCKER_BUILD="false"; fi
+    - echo $DOCKER_BUILD
```

Is meant to stay here? If yes, I'd prefer:
```suggestion
    - echo DOCKER_BUILD=$DOCKER_BUILD
```

---

Fixed and moved it into `if` block. Also, turned out that `"$TRAVIS_PULL_REQUEST" = "true"` was a bad idea (for PRs `TRAVIS_PULL_REQUEST` is a number of that PR, not "true"  🙈 ). Fixed that one too.